web.coffee

```diff
@@ -122,6 +123,9 @@ app.get '/', (request, response) ->
                         for month it must evenly divide 12, and for day and year it
                         can only be 1.
                     </li>
+                    <li>
+                        <em>charset</em>: xml charset <a href="https://github.com/ashtuchkin/iconv-lite/wiki/Supported-Encodings">support charset</a>
```

Comment:
Please change this message to:

`charset used to encode XML. See  <a href="https://github.com/ashtuchkin/iconv-lite/wiki/Supported-Encodings">here for supported charsets</a>.`